### ISSUE TITLE
EMD (NCEM) io_plugin no longer imports mettadata

### ISSUE THREAD

**Issue by erh3cq** [hyperspy/hyperspy#2614]
In previous versions loading a EMD (NCEM) file would load the images and metadata associated with the hdf5 file.  Now, either a dataset_path or stack_group needs to be specified for the data to load the signal and will not load the metadata regardless.
For example:
In the below image I am showing the emd file inspected in EMD viewer.  It is a single (4|4096,4096) image stack with plenty of metadata describing the scope, navigation axis, and signal axis.  Loading the signal without kwargs stacks two (|4) signals. Loading with `stack_group=False` loads two signals, one with he correct (|4,4096,4096) and some unkown (|4) with data that does not make and sense.  Loading with `dataset_path='/data/drift/data'` points to the correct data path.  None of the above load the metadata.
![image](https://user-images.githubusercontent.com/25358789/103419039-449c4380-4b5f-11eb-8e5e-6ed7912e0761.png)
![image](https://user-images.githubusercontent.com/25358789/103419021-2fbfb000-4b5f-11eb-95a0-3e2572d36a86.png)

**Comment by ericpre** [comment]
Do you have a file to share? How was the file generated?

**Comment by erh3cq** [comment]
Here is a file link: [https://virginia.box.com/s/hawao6fqk3w4xgn8xsfngujczgnf77yt](https://virginia.box.com/s/hawao6fqk3w4xgn8xsfngujczgnf77yt)
The data was acquired on a Themis via a python script supplied by NCEM.  It loads in the EMD viewer that NCEM created.

**Comment by ericpre** [comment]
This file contains two datasets in the same group and the function to find the path of the dataset wasn't happy about it. I suspect this dataset should be an attribute instead of a dataset because it simply contain the acquisition time. The file doesn't say which file version it is.
In any case, this is fixed in #2673.

An alternative is to specify the path:
```python
import hyperspy.api as hs

s = hs.load('Rotation_0001.emd', dataset_path='data/drift/data')
<BaseSignal, title: drift, dimensions: (|4, 4096, 4096)>
```

and the `metadata` are in the `original_metadata` attribute:
```python
s.original_metadata
├── EMD_version = None
├── microscope
│   ├── binning = 1
│   ├── camera length = 0.11499999999999999
│   ├── convergence angle = 0.02501599999999996
│   ├── defocus = -2.1499445052725595e-06
│   ├── dwell time = 2.5e-07
│   ├── high tension = 200000.0
│   ├── magnification = 1800000.0
│   ├── max binning = 4096
│   ├── name = FEI
│   └── spot size = 8
├── sample
│   └── sample name = 
└── user
    └── user name = 
```
